### PR TITLE
Ballad Lint and Test Action (WIP)

### PR DESCRIPTION
**Pull request by BD103**
Finally adding Continuous Integration! I have designed a Python Package that simulates installing from a `poetry.lock` file. [It's called Ballad.](https://github.com/BD103/Ballad) Here's what I did:

- Created standard Github Action
- Added linter with Black, Isort, and Flake8
- **It actually works!**
- Only runs for the master branch (which can be changed)
- Because it's my project, I can add features that are requested

### CODE REVIEW

**Comment by JBYT27** [comment]
@BD103, is Ballad basically like pip? And I'm super sorry for not contributing on this much, I'm just busy with other things and I don't exactly know how to contribute that much yet with this kind of stuff.

**Comment by BD103** [comment]
Ballad is a program that simulates the installation process of Poetry. Because I have issues with the CI, this is a workaround that should do the same thing. We'll just have to see if it works!

**Comment by BD103** [comment]
@darkdarcool Oop forgot to request review :^

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Lint
```
Comment (darkdarcool) [inline .github/workflows/lint.yml:1]: Change the name, I don't like `lint`

**Review by darkdarcool — CHANGES_REQUESTED** [review]
LGTM! But fix the name, I really hate it.

**Comment by darkdarcool** [comment]
also @BD103, I think I'll just give reviews on work, and testing is something I'm good at, and maybe I could do the talk leaderboard part after you do user part to see how I would do it?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (BD103) [reply .github/workflows/lint.yml:1]: What name to you suggest?

**Comment by BD103** [comment]
> also @BD103, I think I'll just give reviews on work, and testing is something I'm good at, and maybe I could do the talk leaderboard part after you do user part to see how I would do it?

Sure, that works for me. Talk to RayhanADev about the queries necessary to get all that information.

**Comment by darkdarcool** [comment]
I did, but I'll probably talk to him again, and I'll tinker with graphql every now and then. is there anything that you need me to do now tho?

**Comment by BD103** [comment]
Nope, thanks!

**Comment by darkdarcool** [comment]
ok, noice! Make sure to tell me when to come in and work!

**Comment by darkdarcool** [comment]
Also @BD103, should I add myself to the readme of people who are contributors, or not? Just asking because I just saw.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (darkdarcool) [reply .github/workflows/lint.yml:1]: I like the name `Ballad-Test`, as it seems proper as we're using your module that is SUPER helpful.

**Review by darkdarcool — APPROVED** [review]
Looks Good!

**Review by JBYT27 — APPROVED** [review]
Approving

**Comment by BD103** [comment]
@RayhanADev please approve when you have a chance.

**Comment by RayhanADev** [comment]
> @RayhanADev please approve when you have a chance.

Sorry my bad, I’ll take a look but for future PR’s this week don’t count on an immediate response I’ll be studying until Thursday ;D

**Review by RayhanADev — APPROVED** [review]
LGTM

**Comment by darkdarcool** [comment]
@BD103, @RayhanADev gave the green light, we can merge!

**Comment by BD103** [comment]
Ok, good luck with the exams! Merging.

**Comment by darkdarcool** [comment]
Noice

**Comment by JBYT27** [comment]
Yas

**Pull request merged**